.travis.yml

```diff
@@ -34,8 +34,20 @@ matrix:
     script:
     - yarn jest --outputFile test-results.json --json --runInBand
     - yarn run link
-    - echo "This is the real `danger ci` run on this repo"
-    - DEBUG="*" danger ci --verbose
+
+    # If the PR is odd then run using issue commenting, else use checks
```

Comment:
✨✨ a/b testing ✨✨

Reply:
Thanks man, yeah, I was thinking for a while on how I can ensure this is regularly integrated properly - love this hack, I've not yet got it working but I will get it eventually